pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -87,6 +87,9 @@ type StorageClusterStatus struct {
 	// +optional
 	FailureDomain string `json:"failureDomain,omitempty"`
 
+	// StorageClusterVersion provides the version of the storage cluster
+	StorageClusterVersion string `json:"storageClusterVersion,omitempty"`
```

Comment:
```suggestion
	Version string `json:"version,omitempty"`
```

Reply:
Updated all the files... The CRDs should be updated as we run `generate-k8s` and `generate-openapi` commands, right?